--- simscan-1.4.0/configure.in	2013-11-11 14:21:09.096000007 -0700
+++ simscan-1.4.0-patched/configure.in	2013-11-11 14:23:17.450011124 -0700
@@ -592,7 +592,7 @@
         ])
 
         spamass=""
-        for f in $SA_PATH /usr/bin/spamassassin /usr/local/bin/spamassassin
+        for f in /usr/bin/spamassassin /usr/local/bin/spamassassin
         do
           if test -f $f
           then
@@ -600,6 +600,9 @@
             break;
           fi
         done
+        if test "$SA_PATH" != ""; then
+          spamass=$SA_PATH
+        fi
   
         if test "$spamass" = ""
         then
--- simscan-1.4.0/configure	2013-11-11 14:21:09.103000007 -0700
+++ simscan-1.4.0-patched/configure	2013-11-11 14:25:39.736973404 -0700
@@ -4644,7 +4644,7 @@
 fi;
 
         spamass=""
-        for f in $SA_PATH /usr/bin/spamassassin /usr/local/bin/spamassassin
+        for f in /usr/bin/spamassassin /usr/local/bin/spamassassin
         do
           if test -f $f
           then
@@ -4652,6 +4652,9 @@
             break;
           fi
         done
+        if test "$SA_PATH" != ""; then
+          spamass=$SA_PATH
+        fi
 
         if test "$spamass" = ""
         then
